dashboard/src/components/ui/skeletonCenter.tsx

```diff
@@ -0,0 +1,16 @@
+import type { ReactNode } from 'react';
+import { Skeleton as SkeletonComponent } from '@/components/ui/skeleton'; //to avoid name conflict we rename the import
+
+type SkeletonProps = {
+  children: ReactNode;
+};
+
+const Skeleton = ({ children }: SkeletonProps) => {
```

Comment:
great job here, two things

the `ui` folder is used just for components that have code generated from `shadcn`, move this one level up to the components folder

also, react components file use PascalCase name, it shuold be `SkeletonCenter`, and it is a common lint rule to have the file name the same as the default export, so the file could be named `Skeleton.tsx`

take a look at the `components` folder and see if you see a pattern there